appvalidator/specs/webapps.py

```diff
@@ -318,7 +317,7 @@ class WebappSpec(Spec):
                     "app://*.gaiamobile.org/manifest.webapp": {
                         "expected_type": types.StringTypes,
                         "not_empty": True,
-                        "values": ["2.2", "3.0"]
+                        "value_matches": r"\d+.\d+(?:-\w+)?"
```

Comment:
How about just `r"\d+.\d+(-\w+)?"` ? I don't think you need a non-capturing group, you are not doing anything with it, just testing for a match.